paper-input-container.html

```diff
@@ -277,6 +279,20 @@
         @apply(--paper-input-container-input);
       }
 
+	  .input-content ::content input:focus,
```

Comment:
We can maybe use a computed class, then, like we do for the underline and friends? `[[_computeUnderlineClass(focused,invalid)]]`